.travis.yml

```diff
@@ -947,23 +947,23 @@ matrix:
         - *py27_linux_test_config_env
         - CACHE_NAME=linuxunittests.py27
       script:
-        - ./build-support/bin/ci.sh -2lp
+        - travis_wait 50 ./build-support/bin/ci.sh -2lp
```

Comment:
Frustrating UX experience of using `travis_wait`: the resulting `std{out,err}` will be stripped off any color. We normally print those in red upon test failures to make it easier to scan.

We won't be able to remove `travis_wait` until improving the performance via https://github.com/pantsbuild/pants/issues/7795 or via remoting.